### ISSUE TITLE
generate controller shows warning

### ISSUE THREAD

**Issue by kenjis** [fuel/oil#76]
```
$ php oil g controller form action1 action2
```

Warning - file_get_contents(...fuel/packages/oil/views/crud/template.php): failed to open stream: No such file or directory in PKGPATH/oil/classes/generate.php on line 321

**Comment by WanWizard** [comment]
Which version of FuelPHP? And the Oil package?

**Comment by kenjis** [comment]
1.1/develop from github.

**Comment by WanWizard** [comment]
@philsturgeon: for you?

**Comment by philsturgeon** [comment]
It's moved. Is your submodule up to date?

**Comment by kenjis** [comment]
After updating submodule, there is no error.
Sorry to bother you.

**Comment by danielhunt** [comment]
I'm getting this exact error, using the current 1.1 zip from the main website

**Comment by WanWizard** [comment]
It indeed seems to be broken.

The view method is called like this:

```
static::views($args, 'crud', false);
```

In that method, the template is loaded like this:

```
static::create($app_template, file_get_contents(PKGPATH.'oil/views/'.$subfolder.'/template.php'), 'view');
```

which translates to oil/views/crud/template.php, which indeed does not exist (missing 'scaffold' or 'admin').

@philsturgeon, any comments?

**Comment by philsturgeon** [comment]
Correct, looks like the controller method is not updated in line with other scaffolding updates.

Really we need to reuse the same logic in scaffolding and plain generation, seems weird to have different code doing basically the same thing!

Emailing from my iPhone 4S like a BOSS!

**Comment by WanWizard** [comment]
Ok, labeled it as a bug, and took the liberty of assigning you to this issue... ;)

**Comment by tomas-vccp** [comment]
I'm still getting this error on a brand new install (with oil) today.

**Comment by WanWizard** [comment]
Install of what FuelPHP version? It was fixed after the 1.1 release, so it is only available in 1.1/develop.

**Comment by tomas-vccp** [comment]
I just did 

``` shell
curl get.fuelphp.com/oil | sh
```

and

``` shell
oil create <my-project>
oil g controller home login logout index
```

so I'm assuming it's the latest 

p.s. changelog shows 1.1

**Comment by WanWizard** [comment]
That will get you release 1.1 (which corresponds to the 1.1/master repository here). 

This issue has been fixed after that version was released, so either upgrade to 1.1/develop (download from there, and for both the core and all packages used) or wait for the next release.
